### PR TITLE
Feature/content

### PR DESCRIPTION
**Pull request by Roamer15**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,402 @@
+* {
+  padding: 0;
+  margin: 0;
```
Comment (nganifaith) [inline css/style.css:8]: - [x] apply the style box-sizing: border-box

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+}
+
+body {
+  font-family: system-ui, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  background-color: #f9f9f9;
```
Comment (nganifaith) [inline css/style.css:14]: - [x] apply text-align: justify so it applies to all content. If most content as justified

- [x] add line-height property

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+}
+
+nav {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  color: #036bd3;
```
Comment (nganifaith) [inline css/style.css:25]: - [x] add hover property effects

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+  border: 1px solid #036bd3;
+  color: #036bd3;
+  font-size: 10px;
+  border-radius: 0.8rem;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+header {
+  background-image: linear-gradient(115deg, #1a1a1a80, #0d0d0e70), url("../assets/her-one.png");
```
Comment (nganifaith) [inline css/style.css]: - [ ] background should have a transparency ratio. 
- [x] text justify,  font-style bold
- [x] increase the line height

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+    background-color: transparent;
+  }
+
+  a {
+    color: #f9f9f9;
+  }
+}
+
+h2 {
+  font-family: "Courier New", Courier, monospace;
```
Comment (nganifaith) [inline css/style.css]: I see this is the second time you are using this.

- [x] use CSS variable to declare this font-family as either the primary or secondary font family, So you don't have to write it always

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+  display: block;
+  margin-top: 2em;
+}
+
+.part-two .box-one,
+.part-two .box-two,
+.part-two .box-three,
+.part-two .box-four,
+.part-two .box-five,
+.part-two .box-six {
```
Comment (nganifaith) [inline css/style.css]: - [x] give these element one class in the HTML and use it

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+  border-bottom: 1px solid #8d8d8a;
+}
+
+.list-us {
+  margin-top: 0.5rem;
+}
+
+.list-us .one,
+.list-us .two,
+.list-us .three {
```
Comment (nganifaith) [inline css/style.css]: - [x] give then one class and use it here

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,402 @@
...
+.steps {
+  margin: 0 10px;
+  padding: 1em 0;
+  text-align: center;
+}
+
+.list-steps .consultation,
+.list-steps .package,
+.list-steps .productions,
+.list-steps .order {
```
Comment (nganifaith) [inline css/style.css]: - [x] provide a single class

**Review by nganifaith — CHANGES_REQUESTED** [review]
@Roamer15 Great job so far. I love the effort you've put in. Go through the code and do the changes

File: css/style.css (hunk 4 above)
Comment (Roamer15) [reply css/style.css]: i don't think i understand the transparency ratio part

File: index.html [hunk 9]
```diff
@@ -3,9 +3,302 @@
...
+                    <p>Malesuada dis sollicitudin tristique facilis enim sit erat eleifend nec hedrerit felis</p>
+                    <button><a href="#">GET STARTED</a></button>
+                </div>
+                <div class="box-three">
+                    <img src="./assets/price-tag.png" alt="image">
+                    <p>Malesuada dis sollicitudin tristique facilis enim sit erat eleifend nec hedrerit felis</p>
+                    <button><a href="#">GET STARTED</a></button>
+                </div>
+                <div class="box-four">
+                    <img src="./assets/tv.png" alt="image">
```
Comment (nganifaith) [inline index.html]: - [x] use proper image alt description

File: index.html [hunk 10]
```diff
@@ -3,9 +3,302 @@
...
+                    <p>Malesuada dis sollicitudin tristique facilis enim sit erat eleifend nec hedrerit felis</p>
+                    <button><a href="#">GET STARTED</a></button>
+                </div>
+                <div class="box-five">
+                    <img src="./assets/playback.png" alt="image">
+                    <p>Malesuada dis sollicitudin tristique facilis enim sit erat eleifend nec hedrerit felis</p>
+                    <button><a href="#">GET STARTED</a></button>
+                </div>
+                <div class="box-six">
+                    <img src="./assets/movies.png" alt="image">
```
Comment (nganifaith) [inline index.html]: - [x] use proper alt

File: index.html [hunk 11]
```diff
@@ -3,9 +3,302 @@
...
+                    <div class="one">
+                        <img src="assets/stack.png" alt="imaage">
+                        <div class="paragraph">
+                            <h1>Premium Quality Animations</h1>
+                            <p>Massa cras a congue socisqu vulpate est elementum eleifund quisque egestas</p>
+                        </div>
+                    </div>
+                    <div class="two">
+                        <img src="assets/people.png" alt="imaage">
+                        <div class="paragraph">
```
Comment (nganifaith) [inline index.html:133]: - [x] use proper alt value

File: index.html [hunk 12]
```diff
@@ -3,9 +3,302 @@
...
+                            <h1>Creative team</h1>
+                            <p>Massa cras a congue socisqu vulpate est elementum eleifund quisque egestas</p>
+                        </div>
+                    </div>
+                    <div class="three">
+                        <img src="assets/sound.png" alt="imaage">
```
Comment (nganifaith) [inline index.html]: - [x] use proper alt value

File: index.html [hunk 13]
```diff
@@ -3,9 +3,302 @@
...
+                        <div class="paragraph">
+                            <h1>24/7 Premuim Support</h1>
+                            <p>Massa cras a congue socisqu vulpate est elementum eleifund quisque egestas</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="image-two">
+                <img src="assets/girl.jpg" alt="image">
```
Comment (nganifaith) [inline index.html]: - [x] use proper alt value

File: index.html [hunk 14]
```diff
@@ -3,9 +3,302 @@
...
+            </div>
+        </div>
+        <div class="steps">
+            <div class="stand-image">
+                <img src="assets/stand.jpg" alt="image">
+            </div>
+            <div class="list-steps">
+                <div class="consultation">
+                    <div class="img">
+                        <img src="assets/chat.png" alt="image">
```
Comment (nganifaith) [inline index.html]: - [x] use proper alt value

**Review by nganifaith — CHANGES_REQUESTED** [review]
great job

File: css/style.css (hunk 4 above)
Comment (nganifaith) [reply css/style.css]: look at the original design

**Pull request merged**